sdk/js/src/token_bridge/__tests__/terra2-integration.ts

```diff
@@ -42,6 +50,16 @@ const terraWallet = lcd.wallet(
 );
 const terraWalletAddress = terraWallet.key.accAddress;
 
+const lcdClassic = new LCDClient({
+  URL: !!process.env.CI ? "http://terra-terrad:1317" : "http://localhost:1317",
```

Comment:
Is this the same as TERRA_NODE_URL in file sdk/js/src/token_bridge/__tests__/consts.ts?

Reply:
resolved